### PR TITLE
YLP-2824 Pump logos based on manufacturer

### PR DESCRIPTION
**Pull request by celia-rouis**
## Checklist
- [x] PR name contains the Jira ticket number
- [x] Changelog updated (Jira tickets are correctly sorted)
- [x] Code tested (unit tests and integration tests)
- [x] No console errors/warnings in integration tests
- [x] No translation files updated (except for English files)
- [x] Code formatted on every file
- [x] Copyright added on every file
- [x] End-to-end tests successful on CI

### CODE REVIEW

**Review by TimCanu — APPROVED** [review]
I did not

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +13,7 @@ It is based on Tidepool Blip 1.27.
 - YLP-2586 Display height in device parameters
 - YLP-2724 Filter on weekday for all data types where the information (weekday) is available
 - YLP-2749 Use a branded Verify Email result page instead of the Auth0 one
+- YLP-2824 Display pump logos based on manufacturer
```
[the remark this answers is not in the document]
Comment (celia-rouis) [reply CHANGELOG.md:16]: Yes 😬 Thanks!

**Pull request merged**